app/(afterLogin)/(payroll)/employee-information/[id]/page.tsx

```
@@ -246,375 +248,406 @@ const EmployeeProfile = () => {
-
-                    {/* Total Allowance */}
-                    <div>
-                      <div className="flex flex-col w-full gap-4">
-                        <div className=" pl-4 flex justify-between  items-center my-2">
-                          <Text className="text-xl">Employee Allowance</Text>
-                          <Text className="text-xl pr-10">Amount</Text>
-                        </div>
+                    <Divider />
+                    <div className="flex justify-between">
+                      <Title level={4}>
+                        {dayjs(activePayPeriod?.startDate).format('MMMM')} Pay Slip
```

ESLint

[error] Insert `⏎·······················`

(prettier/prettier)